test/lib/heuristics/scheduling_test.rb

```diff
@@ -515,13 +512,10 @@ def test_provide_initial_solution
 
     def test_fix_unfeasible_initial_solution
       vrp = TestHelper.load_vrp(self, fixture_file: 'instance_baleares2')
-      vrp.routes = Marshal.load(File.binread('test/fixtures/formatted_route.bindump')) # rubocop: disable Security/MarshalLoad
+      vrp.routes = [Models::Route.create(vehicle: vrp.vehicles.first, mission_ids: %w[5482 0833 8595 0352 0799 2047 5446 0726 0708], day_index: 0)]
```

Comment:
Thanks! This is a lot better than dumps, now we can see what the significance is :pray: